src/app/routines/routineDetails/RoutineDetails.tsx

```diff
@@ -103,7 +103,13 @@ const RoutineDetails = () => {
         )}
         {}
       </div>
-      <h2>{data?.routine.name}</h2>
+      <div className='routine-header'>
+        <h2>{data?.routine.name}</h2>
+        <div>
+          <p>Created at:</p>
+          <p> {data?.routine.createdAt ? new Date(data.routine.createdAt).toLocaleDateString() : ''}</p>
```

Comment:
better to render <p> conditionally instead of rendering whats inside of <p> conditionally